ui/server/src/test/scala/pl/touk/nussknacker/ui/util/PdfExporterSpec.scala

```diff
@@ -1,51 +1,48 @@
-import pl.touk.nussknacker.restmodel.processdetails.{BaseProcessDetails, ProcessVersion}
-import pl.touk.nussknacker.engine.api.process.ProcessId
 
+import java.io.FileOutputStream
+import java.nio.charset.StandardCharsets
+import java.time.LocalDateTime
 import scala.io.Source
 
 class PdfExporterSpec extends FlatSpec {
 
+  private val history = List(ProcessVersion(1, LocalDateTime.now(), "Zenon Wojciech", Option.empty, List.empty))
+
   it should "export process to " in {
     val process: DisplayableProcess = ProcessConverter.toDisplayable(ProcessCanonizer.canonize(SampleProcess.process), TestProcessingTypes.Streaming)
-    val displayable = process.copy(nodes = process.nodes.map {
+    val displayable: DisplayableProcess = process.copy(nodes = process.nodes.map {
         case a:Filter => a.copy(additionalFields = Some(UserDefinedAdditionalNodeFields(Some("mój wnikliwy komętaż"), None)))
         case a => a
     })
 
-    val details = BaseProcessDetails("My process", "My process", ProcessId(1), 11, true,
-      Some("My fancy description, which is quite, quite, quite looooooooong. \n And it contains maaaany, maaany strange features..."),false, false,
-      TestProcessingTypes.Streaming, "Category 22", LocalDateTime.now(), LocalDateTime.now(), "user", List(), None, None, Some(displayable),
-      List(ProcessVersion(11, LocalDateTime.now(), "Zenon Wojciech", Option.empty, List.empty)), Option.empty
-    )
+    val details = createDetails(displayable)
 
     val activities = ProcessActivity(List(
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
-      Comment(1L, "aa", 11, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now())
+      Comment(1L, "aa", details.processVersionId, "Jakiś taki dziwny ten proces??", "Wacław Wójcik", LocalDateTime.now()),
```

Comment:
Maybe we can make (1 to 29).map(_ => ...)?

Reply:
Ups, I've not seen your comment.. ;)